locale-source/locale/de/LC_MESSAGES/django.po

```diff
@@ -893,24 +893,24 @@ msgid "seconds"
 msgstr "Sekunden"
 
 #: adhocracy4/modules/models.py:225
-msgctxt "abbreviation for day(s)"
+msgctxt "abbreviation \"day(s)\""
 msgid "D"
-msgstr "T"
+msgstr ""
 
 #: adhocracy4/modules/models.py:227
-msgctxt "abbreviation for hour(s)"
+msgctxt "abbreviation \"hour(s)\""
 msgid "H"
-msgstr "S"
+msgstr ""
 
 #: adhocracy4/modules/models.py:229
-msgctxt "abbreviation for minute(s)"
+msgctxt "abbreviation \"minute(s)\""
```

Comment:
This is weird, when I do an install and then do the "makemesages", these explanations are not changed and the translations not lost.

Reply:
Oh, the translations were also lost? Thats not good..